src/AbstractHPolygon.jl

```diff
@@ -109,6 +123,15 @@ function vertices_list(P::AbstractHPolygon{N},
     end
     points[n] = element(intersection(Line(P.constraints[n]),
                                      Line(P.constraints[1])))
+
+    # check if polygon was empty
+    avg = sum(points) / length(points)
+    for c in constraints_list(P)
```

Comment:
note that one could ask for (see [this function](https://github.com/JuliaReach/LazySets.jl/blob/51c891e09cebdc24dfe0ac0ff604ca31a5fc1561/src/AbstractHPolygon.jl#L175)):

```julia
if avg ∉ P
    return Vector{Vector{N}}(undef, 0)
end
```